src/opencl_pbkdf2_hmac_sha512_fmt_plug.c

```diff
@@ -33,9 +33,9 @@ john_register_one(&fmt_opencl_pbkdf2_hmac_sha512);
 #define FORMAT_NAME              "GRUB2 / OS X 10.8+"
 #define ALGORITHM_NAME           "PBKDF2-SHA512 OpenCL"
 #define BINARY_ALIGN             8
-#define SALT_ALIGN               8
 #define PLAINTEXT_LENGTH         110
 #define SALT_SIZE                sizeof(salt_t)
+#define SALT_ALIGN               sizeof(salt_t*)
```

Comment:
I see no pointer inside the struct. I think 8 was actually right, but we can use `sizeof(uint64_t)` for clarity.

Was there any issue that prompted you to make a change here? I wonder if there's something I overlook.

Reply:
> Was there any issue that prompted you to make a change here? I wonder if there's something I overlook.

No I only saw the warning from fde on a 32-bit build, but looked for others that had a fixed 8. This was the only one and like I said I thought it looked wrong.

> I see no pointer inside the struct. I think 8 was actually right, but we can use `sizeof(uint64_t)` for clarity.

Oh, I'm always confused by these things, that's why I shouldn't try to fix these things. You are obviously right.